draft-ietf-quic-v2.md

```diff
@@ -190,6 +190,18 @@ The client MUST NOT send 0-RTT packets using the negotiated version, even after
 processing a packet of that version from the server. Servers can apply original
 version 0-RTT packets to a connection without additional considerations.
 
+# TLS Resumption
+
+TLS session tickets are specific to the QUIC version of the connection that
+provided them. Clients MUST NOT use a session ticket from a QUICv1 connection
+to initiate a QUICv2 connection, or vice versa.
```

Comment:
👍 I considered suggesting that you make this more generic: "Clients MUST only use a ticket for a QUIC version in which the ticket was received", but you aren't speaking for QUIC versions other than v1 and v2 here, so this is right.

Reply:
Aye, there's a pending "considerations for v1" section in the VN draft where this could live, but I'm happy to have this here as well.